[sc-157321] Allow designating `dir` in github action (#294)
launchdarkly/ld-find-code-refs: build/metadata/github-actions/README.md

| baseUri | The base URL of the LaunchDarkly server for this configuration. | `false` | https://app.launchdarkly.com |
| contextLines | The number of context lines above and below a code reference for the job to send to LaunchDarkly. By default, the flag finder will not send any context lines to LaunchDarkly. If < 0, it will send no source code to LaunchDarkly. If 0, it will send only the lines containing flag references. If > 0, it will send that number of context lines above and below the flag reference. You may provide a maximum of 5 context lines. | `false` | 2 |
| debug | Enable verbose debug logging. | `false` | false |

> | dir | Path to existing checkout of the repository. You may use this if your action checkouts multiple repositories. The default value is `$GITHUB_WORKSPACE`. | `false` |  |

Reviewer: ```suggestion
| dir | Path to existing checkout of the repository. You may use this if your action checks out multiple repositories. The default value is `$GITHUB_WORKSPACE`. | `false` |  |
```

| projKey | Key of the LaunchDarkly project associated with this repository. Found under Account Settings -> Projects in the LaunchDarkly dashboard. Cannot be combined with `projects` block in configuration file. | `false` |  |
| repoName | The repository name. Defaults to the current GitHub repository. | `false` |  |
<!-- action-docs-inputs -->

# Additional inputs

> All [command line flags](https://github.com/launchdarkly/ld-find-code-refs/blob/main/docs/CONFIGURATION.md#command-line) are available as environment variables following the "upper snake case" format, with a prefix of `LD_`. For example, the command line option `dir` may be set as an environment variable.

Reviewer: ```suggestion
All [command line flags](https://github.com/launchdarkly/ld-find-code-refs/blob/main/docs/CONFIGURATION.md#command-line) are available as environment variables following the "upper snake case" format, with a prefix of `LD_`. For example, the command line option `dir` may be set as an environment variable using `LD_DIR`.
```

> For example, if your action checks out multiple repositories you may override the default `dir` which is `GITHUB_WORKSPACE`.
> 
> > GITHUB_WORKSPACE - The default working directory on the runner for steps, and the default location of your repository when using the checkout action. For example, /home/runner/work/my-repo-name/my-repo-name.

Reviewer: ```suggestion
The default `dir` is `GITHUB_WORKSPACE`. This is the default working directory on the runner for steps and the default location of your repository when using the checkout action, for example, `/home/runner/work/my-repo-name/my-repo-name`.

If your action checks out multiple repositories, override the default `dir`.
```
If I'm understanding this correctly, there are two ideas here -- describing the default and when you need to override it. Typically the default description goes first, so that readers learn more about the option before they learn when/how to override.
